### DISCUSSION TITLE
Bruno Golden Edition git pull, push

### DISCUSSION THREAD

**Discussion by liketoeatcheese** [usebruno/bruno#2653] [Q&A] [answered]
Hi all, I got a golden edition license and am trying out the git UI. We do have the ability to add and commit file changes. Do we have a feature to pull changes from upstream and push changes to upstream?

**Comment by sanjai0py** [comment, marked as answer]
Hey @liketoeatcheese, we are currently working on making a few final improvements to the feature. The git-push and pull feature is expected to be ready in a couple of weeks. Thank you for supporting Bruno by purchasing the golden edition. We appreciate your patience!

**Reply by liketoeatcheese** [reply]
Thank you for your work :)
